sql/src/test/java/io/crate/planner/operators/WindowAggTest.java

```diff
@@ -66,6 +66,15 @@ public void testTwoWindowFunctionsWithDifferentWindowDefinitionResultsInTwoOpera
         assertThat(plan, isPlan(e.functions(), expectedPlan));
     }
 
+    @Test
+    public void test_window_agg_output_for_select_with_standalone_ref_and_window_func_with_filter() {
+        var plan = plan("SELECT y, AVG(x) FILTER (WHERE x > 1) OVER() FROM t1");
```

Comment:
Would it make sense to adapt the test further so that `FILTER (...` uses  a column that is otherwise not used? (`y` here is in the outputs, `x` is used in the aggregate)

Reply:
I will add a separate test for it. It seems that the plan is not correct for this case, such as `Collect` contains the filter symbol in it :(

Reply:
That should be okay?